### PR TITLE
chore(docs): Add links to help first-time contributors

### PR DESCRIPTION
**Pull request by kentcdodds**
Invitation: https://twitter.com/dignifiedquire/status/696758773067739136

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ If you find a bug in the source code or a mistake in the documentation, you can
 submitting an issue to our [GitHub Repository][github_newissue]. Even better you can submit a Pull Request
 with a fix.
 
+**Working on your first Pull Request?** You can learn how from this *free* series
```
Comment (dignifiedquire) [inline CONTRIBUTING.md]: We will need this on here: https://github.com/karma-runner/karma/blob/master/docs/dev/01-contributing.md as well as that is the front facing page on the website to get started

**Comment by dignifiedquire** [comment]
Thanks!

**Comment by kentcdodds** [comment]
Updated!

**Comment by dignifiedquire** [comment]
Sweet, thanks for doing this!

**Pull request merged**